tests/CacheCompat/CommonCache.Test.MsalNode/NodeScript/index.js

```diff
@@ -0,0 +1,180 @@
+﻿/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+
+/* To run this locally, start with `node index.js --inputPath c:\\temp\\input.json` where input.json contains
+
+{
+    "Scope": "https://graph.microsoft.com/user.read",
+    "CacheFilePath": "C:\\Users\\bogavril\\AppData\\Local\\Temp\\adalcachecompattestdata\\msalCacheV3.binx",
+    "LabUserDatas": [
+        {
+            "Upn": "liu.kang@bogavrilltd.onmicrosoft.com",
```

Comment:
do you want to leave this here?

Reply:
Yes, I wanted to show a real example. The user is in a private tenant and not connected to anything.